dio/lib/src/adapter.dart

```diff
@@ -4,7 +4,8 @@ import 'dart:typed_data';
 import 'package:meta/meta.dart';
 
 import 'adapters/io_adapter.dart'
-    if (dart.library.html) 'adapters/browser_adapter.dart' as adapter;
+    if (dart.library.js_util) 'adapters/web_adapters/web_adapter.dart'
```

Comment:
This should be `dart.library.js_interop` as mentioned here: https://dart.dev/interop/js-interop/package-web#conditional-imports

and everywhere else in the dio package